edx-platform/pearson-theme/lms/templates/courseware/course_about.html

```diff
@@ -11,9 +11,13 @@
 from openedx.core.djangolib.js_utils import js_escaped_string
 from openedx.core.djangolib.markup import clean_dangerous_html, HTML, Text
 from openedx.core.lib.courses import course_image_url
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 
 from six import string_types
 %>
+<%
+button_enroll_bottom = configuration_helpers.get_value('button_enroll_bottom', ' ')
```

Comment:
Since this is a boolean value, the default should be False instead of ' '. @amalbas